### ISSUE TITLE
 ESC key fails to close DatePicker dialog in WebView when using JAWS

### ISSUE THREAD

**Issue by ioana-gln** [FreedomScientific/standards-support#832]
**Summary**
Pressing the ESC key for the first time in the expanded date picker within WebView triggers the activation of the virtual PC cursor mode, and a subsequent press closes the dialog when utilizing the JAWS screen reader.

Open WebView2 and navigate to https://65fa8a6751742e21b1bc65bd-tvzgszrenx.chromatic.com/?path=/story/form-dateinput--date-input-story
Focus the date input field
Press ArrowDown to open date picker 
Press Esc to close date picker
**Expected result**
Pressing the ESC key for the first time in the expanded date picker within WebView closes the date picker

**Actual result**
Pressing the ESC key for the first time in the expanded date picker within WebView triggers the activation of the virtual PC cursor mode, and a subsequent press closes the popover when utilizing the JAWS screen reader.

**Example**
https://65fa8a6751742e21b1bc65bd-tvzgszrenx.chromatic.com/?path=/story/form-dateinput--date-input-story
https://www.w3.org/WAI/ARIA/apg/patterns/combobox/examples/combobox-datepicker/
**Additional Information**
Works with NVDA 2024.1
Works with JAWS when used in normal browser. 

**JAWS version and build number**
Version 2024.2403.3

**Operating System and version**
Windows 11

**Browser and version:**
WebView2 123.0.2420.97
